packages/models-library/src/models_library/api_schemas_webserver/groups.py

```diff
@@ -383,3 +385,17 @@ class GroupUserUpdate(InputSchema):
             }
         }
     )
+
+
+class ProjectShare(InputSchema):
```

Comment:
same thought here.
Why is this in the groups.py module?
To me this does not connect. A group is a group of users. What is this ProjectShare and ProjectShareAccepted doing here?